CarpetLib/src/interpolate_eno_3d_3tl.cc

```diff
@@ -129,15 +129,14 @@ void interpolate_eno_3d_3tl(
   bool const use_12 = t >= min(t1, t2) - eps and t <= max(t1, t2) + eps;
   bool const use_23 = t >= min(t2, t3) - eps and t <= max(t2, t3) + eps;
   assert(use_12 or use_23);
-// TODO: Instead of use_12, calculate 3 coefficents that perform
-// the desired 2-point interpolation, which would avoid the if
-// statement in the loop, simplifying the code.
+  // TODO: Instead of use_12, calculate 3 coefficents that perform
+  // the desired 2-point interpolation, which would avoid the if
+  // statement in the loop, simplifying the code.
 
-// Loop over region
-#pragma omp parallel
+  // Loop over region
+  // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.